resources/js/Components/Form/CampaignForm.tsx

```diff
@@ -59,7 +61,26 @@ const CampaignForm = ({
                   adventure: editData?.adventure || undefined,
               }
 
-    const {data, setData, errors, put, post} = useForm<CampaignFormDataType>(CAMPAIGN_INITIAL_VALUE)
+    const {data, setData, errors, clearErrors, put, post, wasSuccessful} =
```

Comment:
can you also add `processing` as well for loading button here?

Reply:
on it!